### ISSUE TITLE
WCAG 4.1.2 test 9: false positive

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#89]
[This very simple file](https://github.com/veraPDF/veraPDF-wcag-algs/files/7389188/lijst-arial.pdf) fails **WCAG 4.1.2 test 9** ('_Missing Table element(s)_'). However, it is just a couple of paragraphs and a correctly tagged list. There is nothing in this document that even remotely looks like a table.

Can you check this false positive out? Perhaps the algorithm can avoid this error when it  sees there is a correctly tagged list in the part it perceives as a table.

**Comment by jzuidweg** [comment]
@bdoubrov , [This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7736330/test-document-3.pdf) and 
[this document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7736332/test-document-6.pdf) also raise false positives for WCAG 4.1.2 test 9. Perhaps this helps you fix the issue.
